File: Python/1066 - Pares, Ímpares, Positivos e Negativos.py
valores = [int(input()) for _ in range(5)]

pares = sum(1 for v in valores if v % 2 == 0)
impares = sum(1 for v in valores if v % 2 != 0)
positivos = sum(1 for v in valores if v > 0)
negativos = sum(1 for v in valores if v < 0)

print(f"{pares} valor(es) par(es)")
print(f"{impares} valor(es) impar(es)")
print(f"{positivos} valor(es) positivo(s)")
print(f"{negativos} valor(es) negativo(s)")

File: JavaScript/1066 - Pares, Ímpares, Positivos e Negativos.js
var valores = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').map(Number);
var pares = valores.filter(v => v % 2 === 0).length;
var impares = valores.filter(v => v % 2 !== 0).length;
var positivos = valores.filter(v => v > 0).length;
var negativos = valores.filter(v => v < 0).length;

console.log(`${pares} valor(es) par(es)`);
console.log(`${impares} valor(es) impar(es)`);
console.log(`${positivos} valor(es) positivo(s)`);
console.log(`${negativos} valor(es) negativo(s)`);